gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,35 @@
+from oslo_config import cfg
+
+OPTS = [
+    cfg.IntOpt(
+        'workers',
+        default=1,
+        help=_('#of workers to create.')
+    ),
+    cfg.StrOpt(
+        'RpcLoadBalancer',
```

Comment:
Actually the idea was if we could implement any good loadbalancing algorithm for distribution of events to worker process, we should be able to select it from this config options v/s the code change in framework. New loadbalancer algo can be implemented in core/lb.py and this conf option is used to make code point to that loadbalancer. Ok will make its name consistent and change its name to "event_loadbalancer"